### PR TITLE
Adding footer and scroll to top button on every page

### PR DESCRIPTION
**Pull request by s2sharpit**
## Related Issue

- Adding Footer and ScrollToTop Button to all pages of the website.

Closes: #351 

### Describe the changes you've made
Moved footer code from the landing page to another component named Footer.
Added Footer Component into `layout.tsx`.
Moved ScrollToTop Component from landing page and added to `layout.tsx`.
Changed Section tag `<section>` into footer tag `<footer>`

I moved the footer and scrollToTop button into `layout.tsx` so that it can be seen on every page of website.
Changed section tag `<section>` to footer tag `<footer>` for footer section.

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

|      Original       |      Updated       |
| :-----------------: | :----------------: |
| original screenshot | updated screenshot |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by darkterminal — APPROVED** [review]
:zap: L A T M ! ! !

**Review by mkubdev — APPROVED** [review]
LGTM 🚀

File: components/main-nav.tsx [hunk 1]
```diff
@@ -41,17 +41,17 @@ export function MainNav({ items }: MainNavProps) {
...
+            <ul className="flex flex-1 list-none flex-col items-start justify-end gap-4">
               {items?.map(
                 (item, index) =>
                   item.href && (
                     <Link
                       key={index}
                       href={item.href}
-                      className={`font-poppins font-medium cursor-pointer text-[16px] 
-                  }`}
+                      className={`font-poppins } cursor-pointer text-[16px]
```
Comment (mkubdev) [inline components/main-nav.tsx:53]:
```suggestion
                      className={`font-poppins cursor-pointer text-[16px] font-medium`}
```

File: components/main-nav.tsx [hunk 2]
```diff
@@ -41,17 +41,17 @@ export function MainNav({ items }: MainNavProps) {
...
               {items?.map(
                 (item, index) =>
                   item.href && (
                     <Link
                       key={index}
                       href={item.href}
-                      className={`font-poppins font-medium cursor-pointer text-[16px] 
-                  }`}
+                      className={`font-poppins } cursor-pointer text-[16px] 
+                  font-medium`}
```
Comment (mkubdev) [inline components/main-nav.tsx:54]:
```suggestion

```

**Review by mkubdev — COMMENTED** [review]
LGTM! 👌 Thanks a lot for your contribution. 😁

**Review by mkubdev — APPROVED** [review]
🚀 LGTM

**Review by vinzvinci — APPROVED** [review]
LGTM!

**Pull request merged**